src/OpenLoco/Company.h

```diff
@@ -46,6 +47,41 @@ namespace OpenLoco
         surveyingLandscape,
     };
 
+    class OwnerStatus
+    {
+        int16_t data[2];
+
+    public:
+        OwnerStatus()
+        {
+            data[0] = -1;
+            data[1] = 0;
+        }
+        OwnerStatus(EntityId id)
+        {
+            data[0] = -2;
+            data[1] = enumValue(id);
+        }
+        OwnerStatus(const Map::Pos2& pos)
+        {
+            data[0] = pos.x;
+            data[1] = pos.y;
+        }
+
+        constexpr bool isEntity() const { return data[0] == -2; }
```

Comment:
what does the `-2` represent? if we can put it in a const variable with a name that would be great

Reply:
I'm not sure. `-2` was the value for `ax` when using the entityId for `cx` in the code that hooked to the original function. I guess it is used as an invalid / unused marker for that register. What do you suggest the name should be? Should it be a const variable inside the OwnerStatus class?